style.css

```diff
@@ -0,0 +1,99 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+}
+
+.header {
+  display: flexbox;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  background-color: #fff;
+  padding-top: 30px;
+  color: #6070ff;
+  margin: 15px;
+}
+
+.headline-section {
+  display: flexbox;
```

Comment:
- Good job trying to align this section using the flex display property but I think you meant to use flex, not flexbox. Please fix this 👍 👍

Reply:
Okay I have changed everything as per your instructions.